src/main/java/edu/harvard/iq/dataverse/workflows/review/Comment.java

```diff
@@ -0,0 +1,56 @@
+package edu.harvard.iq.dataverse.workflows.review;
+
+import edu.harvard.iq.dataverse.DatasetVersion;
+import java.io.Serializable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+
+@Entity
+public class Comment implements Serializable {
```

Comment:
Let's add a foreign key to the Authenticated user who leaves the comment. (Again we want to err on the side of having too much info we may not need, versus not having something we'll need later).